Jenkinsfile

```diff
@@ -130,7 +130,7 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
             'BUILD_SMPE'      : (params.BUILD_SMPE ? 'yes' : ''),
             'KEEP_TEMP_FOLDER': (params.KEEP_TEMP_FOLDER ? 'yes' : '')
           ],
-          extraFiles          : (params.BUILD_SMPE ? 'zowe-smpe.zip,fmid.zip,pd.htm,smpe-promote.tar,smpe-build-logs.pax.Z,rename-back.sh' : ''),
+          extraFiles          : (params.BUILD_SMPE ? 'zowe-smpe.zip,fmid.zip,pd.htm,smpe-promote.tar,smpe-build-logs.pax.Z,rename-back.sh,RefRuntimeHash.txt,HashFiles.class' : ''),
```

Comment:
I believe your new changes have put RefRuntimeHash.txt and HashFiles.class into pax file. So I believe we don't need to change this file any more.

Reply:
ok